store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java

```diff
@@ -95,6 +95,7 @@ public Key getKeyFromEntity(final Entity entity) {
         return new Key(rowKey, columnFamily, columnQualifier, columnVisibility, timeStamp);
     }
 
+    @SuppressWarnings("Convert2streamapi")
```

Comment:
It would be useful to include a justification in your SuppressWarnings annotation.